### PR TITLE
Automate application deployment and other things

### PR DESCRIPTION
**Pull request by nachandr**
Added the following through this PR:
1)Automation of application deployment on source cluster
2)On source cluster, verify that application is running before migration
3)On target cluster, verify that application is running after migration

### CODE REVIEW

File: cypress/integration/tests/indirect_migration_using_S3.test.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import { noVerifyCopyPlanData, verifyCopyPlanData, directPvPlanData, verifyCopydirectPvPlan,
   directImagePlanData, directImagePvPlan, indirectMultipleProjects, directMultipleProjects, changeTargetNamespace,
   IndirectChangeTargetNamespace } from './cluster_config';
-import { login } from '../../utils/utils';
+import { login, setup_source_cluster, cleanup_source_cluster, post_migration_verification } from '../../utils/utils';
```
Comment (nitishSr) [inline cypress/integration/tests/indirect_migration_using_S3.test.ts:4]: wrong import - "post_migration_verification_on_target"

File: cypress/integration/tests/indirect_migration_using_S3.test.ts [hunk 2]
```diff
@@ -31,6 +31,7 @@ describe('Automated tests to do direct and indirect migrations using Amazon S3 u
     const [Data, migrationType] = $type;
 
     it(`${migrationType}`, () => {
+      setup_source_cluster();
```
Comment (nitishSr) [inline cypress/integration/tests/indirect_migration_using_S3.test.ts:34]: Can we have it in before() hook

File: cypress/integration/tests/indirect_migration_using_S3.test.ts [hunk 3]
```diff
@@ -39,6 +40,8 @@ describe('Automated tests to do direct and indirect migrations using Amazon S3 u
         plan.execute(Data);
       }
       plan.delete(Data);
+      cleanup_source_cluster();
+      post_migration_verification();
```
Comment (nitishSr) [inline cypress/integration/tests/indirect_migration_using_S3.test.ts:44]: These two can be moved to after() hook

**Review by nitishSr — CHANGES_REQUESTED** [review]
Can we also have same changes of calling setup_cluster, cleanup cluster methods in migration_pipeline_status_test

**Comment by nachandr** [comment]
All the enhancements were merged through https://github.com/konveyor/crane-ui-tests/pull/14 . So, closing this PR.

**Pull request closed without merging**